javascripts/src/mobile-nav.js

```diff
@@ -14,6 +14,16 @@
     this.setup();
   }
 
+  function toggleItem($toggler, $parent, $icon) {
```

Comment:
move this function back into your class:

`MobileMenu.prototype.toggleItem = toggleItem($toggler, $parent, $icon) { ... }`